app/views/external_users/advocates/hardship_claims/_miscellaneous_fees_form_step.html.haml

```diff
@@ -0,0 +1,12 @@
+- locale_scope = 'external_users.claims.misc_fees'
+
+= content_for :page_title, flush: true do
+  = t('.page_title')
+
+= render partial: 'external_users/claims/fees_shared_header', locals: {page_header: t('page_header', scope: locale_scope), page_hint: t('page_hint', scope: locale_scope), fees_calculator_html: ''}
```

Comment:
Note: line seems really long.

Reply:
unfortunately `Illegal nesting: nesting within plain text is illegal.` there is probably  away to tidy this up  - same goes for existign case type select list below this. 

you could try a PR